packages/devops/scripts/deployment-vm-app/install.sh

```
@@ -34,6 +34,13 @@ nvm use
 corepack enable yarn
 yarn install --immutable
 
+if ! $bw --version; then
```

Shellcheck (0.10.0)

[warning] bw is referenced but not assigned.

(SC2154)